templates/nas-grp-base.html.twig

```diff
@@ -159,6 +159,10 @@
         });
     </script>
 
+    <script>
+        window.PELAGOS_TEMPLATE_PROPS = {"showSearchTextBlock": false};
```

Comment:
Needs to be set to TRUE in the griidc template

Reply:
yes, currently the search page doesn't show at all under the griidc template.

Reply:
I'm just gonna flip it around.